### PR TITLE
Fix for query editor toolbar actions to show correctly based on file type

### PR DESCRIPTION
**Pull request by gupta1**
This PR fixes #
Fix for query editor toolbar actions to show correctly based on file type when not connected

### CODE REVIEW

File: src/sql/workbench/contrib/query/browser/queryEditor.ts [hunk 1]
```diff
@@ -260,9 +260,10 @@ export class QueryEditor extends BaseEditor {
 		let content: ITaskbarContent[];
 		const previewFeaturesEnabled = this.configurationService.getValue('workbench')['enablePreviewFeatures'];
 		let connectionProfile = this.connectionManagementService.getConnectionProfile(this.input?.uri);
+		let fileExtension = this.input?.uri.substr(this.input?.uri.lastIndexOf('.') + 1);
 
 		// TODO: Make it more generic, some way for extensions to register the commands it supports
-		if (connectionProfile?.providerName === 'KUSTO') {
+		if (connectionProfile?.providerName === 'KUSTO' || fileExtension === 'kql' || fileExtension === 'kusto') {
```
Comment (Charles-Gagnon) [inline src/sql/workbench/contrib/query/browser/queryEditor.ts:266]: You should use https://github.com/microsoft/azuredatastudio/blob/main/src/vs/editor/common/services/modeService.ts#L40 to get the supported extensions so you don't have to update this if you add more later on.

File: src/sql/workbench/contrib/query/browser/queryEditor.ts [hunk 2]
```diff
@@ -260,9 +260,10 @@ export class QueryEditor extends BaseEditor {
 		let content: ITaskbarContent[];
 		const previewFeaturesEnabled = this.configurationService.getValue('workbench')['enablePreviewFeatures'];
 		let connectionProfile = this.connectionManagementService.getConnectionProfile(this.input?.uri);
+		let fileExtension = this.input?.uri.substr(this.input?.uri.lastIndexOf('.') + 1);
```
Comment (Charles-Gagnon) [inline src/sql/workbench/contrib/query/browser/queryEditor.ts:263]: [path.extname](https://github.com/microsoft/azuredatastudio/blob/main/src/vs/base/common/path.ts#L183)

File: src/sql/workbench/contrib/query/browser/queryEditor.ts (hunk 1 above)
Comment (gupta1) [reply src/sql/workbench/contrib/query/browser/queryEditor.ts:266]: Makes sense. Made this change

**Comment by gupta1** [comment]
> Did you verify that this works correctly if you switch the languages of an opened document? (i.e. switching from sql to kql causes the actions to be updated correctly)

I see one issue that xml as well as kusto have same supported extension ".csl". Should I remove ".csl" from kusto supported extensions for now?

**Pull request merged**